admins_guide/adding_users_to_group.py

```diff
@@ -0,0 +1,54 @@
+import os
+import requests
+
+"""
+**DISCLAIMER:** 
+	Please note that this code is for educational purposes only. 
+	It is not intended to be run directly in production. 
+	This is provided on a best effort basis. 
+	Please make sure the code you run does what you expect it to do.
+"""
+
+"""
+DESCRIPTION
+	Adding new members to your VirusTotal group by their email address.
+REQUIREMENTS
+	Admin privileges -> API key as VT_APIKEY environment variable (os.environ['VT_APIKEY'])
+	Update GROUP_ID variable. Check out your group ID here: https://www.virustotal.com/gui/group/virustotal/users 
```

Comment:
Maybe I would check my group name by either going to the UI or calling this API call:
```
GET/api/v3/users/{username}/relationships/groups
```

Reply:
Changing this